<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_1">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt 'KoPubWorld돋움체_Pro Bold';color:red;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;25,913,809&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt 'KoPubWorld돋움체_Pro Medium';background-color:red;color:white</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;+5.17%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;color:gray;
</string>
       </property>
       <property name="text">
        <string>거래량(24H)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;color:gray;
</string>
       </property>
       <property name="text">
        <string>고가(당일)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;</string>
       </property>
       <property name="text">
        <string>14,963.8166BTC</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;color:gray;
</string>
       </property>
       <property name="text">
        <string>채결강도(24H)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;color:gray;
</string>
       </property>
       <property name="text">
        <string>거래금액(24H)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;</string>
       </property>
       <property name="text">
        <string>54,298.000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;</string>
       </property>
       <property name="text">
        <string>7,830.4억</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;</string>
       </property>
       <property name="text">
        <string>93.28%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;color:gray;
</string>
       </property>
       <property name="text">
        <string>전일종가</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;color:gray;
</string>
       </property>
       <property name="text">
        <string>저가(당일)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;</string>
       </property>
       <property name="text">
        <string>49,440.000</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;KoPubWorld돋움체_Pro Bold&quot;;</string>
       </property>
       <property name="text">
        <string>51,403,000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
